roles/is-analytics-dashboard/tasks/main.yml

```diff
@@ -15,47 +15,39 @@
 #----------------------------------------------------------------------------
 ---
   - block:
-    # JDK Distribution
-    - name: Set rpm_java_home
+    # Set installation directory information
+    - name: Installation Information
       set_fact:
-        lib_dir: "{{ rpm_lib_dir }}"
-        java_home: "{{ rpm_lib_dir }}/{{ jdk_name }}"
-      when: ansible_os_family == "RedHat"
+        product_dir: "{{ target }}/is-analytics-dashboard"
+        carbon_home: "{{ target }}/is-analytics-dashboard/{{ product_name }}-{{ product_version }}"
 
-    - name: Set deb_java_home
-      set_fact:
-        lib_dir: "{{ deb_lib_dir }}"
-        java_home: "{{ deb_lib_dir }}/{{ jdk_name }}"
-      when: ansible_os_family == "Debian"
-
-    - name: Create lib directory
+    # WSO2 Distribution
+    - name: Create product directory
       file:
-        path: "{{ lib_dir }}"
+        path: "{{ product_dir }}"
         state: directory
+        mode: 0755
 
-    - name: Unarchive JDK distribution
-      unarchive:
-        src: "{{ product_package_location }}/{{ jdk_name }}.tar.gz"
-        dest: "{{ lib_dir }}"
-
-    # WSO2 Distribution
+    - name: Stop is-analytics-dashboard as a service
+      systemd:
+        name: is-analytics-dashboard
+        state: stopped
 
-    - name: Create distribution path
+    - name: Remove existing setup
       file:
-        path: "{{ distribution_path }}"
-        state: directory
-        mode: 0755
+        path: "{{ carbon_home }}"
+        state: absent
 
     - name: Unzip Analytics Dashboard Package
       unarchive:
-        src: "{{ product_package_location }}/{{ product_name }}-{{ product_version }}.zip"
-        dest: "{{ distribution_path }}"
+        src: "{{ product_package_location }}/packs/{{ product_name }}-{{ product_version }}.zip"
+        dest: "{{ product_dir }}"
         mode: u=rw,g=r,o=r
 
     - name: Copy JDBC Driver
       copy:
-       src: "files/{{ jdbc_driver }}"
-       dest: "{{ install_path }}/lib"
+       src: "files/lib/{{ jdbc_driver }}"
```

Comment:
Since we are running using the H2 database by default, shall we comment these?

Reply:
Fixed https://github.com/wso2/ansible-is/pull/22/commits/c944191a60781df1922abf9b07619ddb62bf61e7